lib/Internal/Placeholder.php

```diff
@@ -3,7 +3,8 @@
 namespace Amp\Internal;
 
 use Amp\Failure;
-use Interop\Async\{ Loop, Promise };
+use Interop\Async\Loop;
```

Comment:
It's never worth. Group use shouldn't be a thing. It's way harder to read and find the right entries.